vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -40,6 +47,7 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
 
     @Inject lateinit var notificationUtils: NotificationUtils
     @Inject lateinit var locationTracker: LocationTracker
+    @Inject lateinit var session: Session
```

Comment:
is it safe to inject the `session` directly or should we use the `ActiveSessionHolder`? :thinking:

Reply:
Right, not safe here. Done.